File: qlty-cli/tests/lang/cpp/basic.in/parameters.cpp
#include <string>

class Parameters {
public:
    static void f0() {
    }

    static void f1(int dog, int cat) {
    }

    static void f2(int a, int b, int c, int d, int e, int f) {
    }

    static void f3() {
        int foo = bar(1, 2, 3, 4);
    }

    static int bar(int a, int b, int c, int d) {
        return a + b + c + d;
    }
};

File: qlty-cli/tests/lang/vbnet/basic.in/Parameters.vb
Imports System

Public Class Parameters
    Public Shared Sub F0()
    End Sub

    Public Shared Sub F1(dog As Object, cat As Object)
    End Sub

    Public Shared Sub F2(a As Object, b As Object, c As Object, d As Object, e As Object, f As Object)
    End Sub

    Public Shared Sub F3()
        Dim foo As Object = Bar(1, 2, 3, 4)
    End Sub

    Public Shared Function Bar(a As Integer, b As Integer, c As Integer, d As Integer) As Object
        Return New Object()
    End Function
End Class
